L1Trigger/Phase2L1ParticleFlow/interface/APxLinePair.h

```diff
@@ -0,0 +1,196 @@
+#ifndef L1Trigger_Phase2L1ParticleFlow_APxLinePair_h
+#define L1Trigger_Phase2L1ParticleFlow_APxLinePair_h
```

Comment:
Given this is not specific to L1PF in any way, shouldn't it be put elsewhere?
Vladimir, is there a package for trigger utilities of this sort?

Reply:
It's not specific to L1PF, but it is specific to 96-bit track words. This class encodes two 96-bit tracks onto 3 lines of an APxPatternFile. The other 64-bit words (other objects/inputs) will not need to use the APxLinePair code.